### ISSUE TITLE
Please Update This Shader To 1.20

### ISSUE THREAD

**Issue by LilNugget95** [Mcbamboo/bicubic-shaders-be#51]
With the new update, attempting to use the shader results in blocks not loading, and it's just a void. Please update this shader, it is my entire personality on Hive

**Comment by Electrolmusic** [comment]
Yes, entities are still visible. But blocks are invisible after 1.20 Update.
